cinder_plugin/tests/test_volume.py

```diff
@@ -339,4 +340,317 @@ def test_detach(self, wait_until_status_m, cinder_m, nova_m):
             status=volume.VOLUME_STATUS_AVAILABLE,
             num_tries=10,
             timeout=2,
+        )
+
+    def _simple_volume_ctx(self):
+        volume_id = '1234-5678'
+        volume_ctx = cfy_mocks.MockCloudifyContext(
+            node_id="node_id",
+            node_name="node_name",
+            properties={},
+            runtime_properties={
+                OPENSTACK_ID_PROPERTY: volume_id,
+            }
+        )
+        current_ctx.set(volume_ctx)
+        return volume_ctx, volume_id
+
+    @mock.patch('openstack_plugin_common.CinderClientWithSugar')
+    @mock.patch('time.sleep', mock.Mock())
```

Comment:
@0lvin This is also need to be updated in case you removed the time.sleep you need to take care of this as well

CC : @EarthmanT

Reply:
@mabuaisha removed